unified_planning/engines/compilers/grounder.py

```diff
@@ -82,6 +92,14 @@ def __init__(
         self._substituter = Substituter(env)
         self._simplifier = Simplifier(env, problem)
 
+    @property
+    def substituter(self) -> Substituter:
+        return self._substituter
+
+    @property
+    def simplifier(self) -> Simplifier:
+        return self._simplifier
+
```

Comment:
Why is it needed to expose these class attributes?

Reply:
Because they are used outside just to ground 1 expression. I thought about a `ground_expression` method, but it just calls the substituter and simplifier over the expression.